app/main/helpers/supplier_details.py

```diff
@@ -16,6 +16,7 @@ def get_company_details_from_supplier(supplier):
             or
             supplier.get("otherCompanyRegistrationNumber")
         ),
+        "registered_with": "companies_house" if "companiesHouseNumber" in supplier else "unknown",
```

Comment:
Where do we use this?

Reply:
Oh I was going to add a conditional to the Companies House URL but I forgot to add that bit.

Reply:
Added now